docs/paper/dev/api/event-api/chat-event.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/chat-events
+description: How to properly use the new AsyncChatEvent 
+---
+
+# Chat Events
+The chat event has evolved a couple of times over the years. 
```

Comment:
```suggestion
The Chat event has evolved a few times over the years. 
```

Reply:
Why exactly should "Chat" be uppercase here?

Reply:
Looks nicer i think… doesn’t need to be though